packages/stellar-axelar-std-derive/src/storage.rs

```diff
@@ -259,13 +268,15 @@ impl Value {
                 remover: format_ident!("remove_{}_status", ident),
                 try_getter: format_ident!("_"),
                 ttl_extender: format_ident!("extend_{}_ttl", ident),
+                has: format_ident!("has_{}_status", ident),
```

Comment:
this is only being added for value type. The status type has a `is_*` getter already